tests/testthat/test-qenv_within.R

```diff
@@ -131,3 +131,21 @@ testthat::test_that("within run on qenv.error returns the qenv.error as is", {
 
   testthat::expect_identical(qe, qee)
 })
+
+testthat::describe("within run with `=`", {
+  testthat::it("single expression", {
+    q <- qenv()
+    q <- within(q, {
+      i <- 1
```

Comment:
@averissimo test cays that `run with =`. Should we use `=` in the test?
```suggestion
      i = 1
```

Reply:
Thanks! Fixed here 674d872a

Reply:
This was done automatically by styler. It seemed strange to me that the test would have this.